glide.lock

```diff
@@ -110,7 +110,7 @@ imports:
   - name: golang.org/x/sys
     version: d2e65c121b96ff63726fb564ece016e70ec81a44
   - name: github.com/google/go-github
-    version: 9686ff0746200cf521ce225525b421e13b4eac1a
+    version: 0c6788b2a83e5ba928a54acf4572a93b169973e3
```

Comment:
The functions we had for getting team membership were depreciated--this is the commit where they were officially released. I didn't vendor using the newest revision because it looked like it was failing checks / didn't want to inadvertently break anything.

Reply:
but would be be safer with an official release? Is this an official release?

Reply:
Ah I see. I've never really looked into "official releases" of libraries but I suppose I can just update to the most recent release.

Reply:
This new version is from the most recent release: https://github.com/google/go-github/releases/tag/v32.1.0